### PR TITLE
Add script for local deployment

### PR DESCRIPTION
**Pull request by NautiluX**
Add script to allow deployment of the generated artifacts without using the `SyncSet`

### CODE REVIEW

**Comment by lisa** [comment]
(Note: This was split off #137 by request)

**Comment by lisa** [comment]
I tried to test this locally and of course I have the _other_ yq.

**Comment by NautiluX** [comment]
yq is a mess

File: hack/deploy-dev.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This script will break up the selectorsyncset in build/selectorsyncset.yaml
+# and deploy all the objects to the currently logged in cluster
+# for development purposes.
+#
+# example flow:
+# $ export IMG=quay.io/my-user/managed-cluster-validating-webhooks IMAGETAG=latest IMAGE_TAG=latest
```
Comment (NautiluX) [inline hack/deploy-dev.sh:8]:
```suggestion
# $ export IMG=quay.io/my-user/managed-cluster-validating-webhooks IMAGETAG=latest IMAGE_TAG=latest SYNCSET_GENERATOR_IMAGE=golang:1.14
```

File: hack/deploy-dev.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# $ make build-image
```
Comment (NautiluX) [inline hack/deploy-dev.sh:9]:
```suggestion
# $ make syncset
```

**Comment by lisa** [comment]
Closing for forthcoming alternative.

**Pull request closed without merging**